### PR TITLE
Update design of Organisation Index Page

### PR DESCRIPTION
**Pull request by davidgisbey**
## Description 

This page was one of the first pages we ported and isn't inline with the latest patterns we've used. This updates the design to be consistent with the Worldwide org index page.

### Before 

image

### After 

image

## Trello card

https://trello.com/c/S2cUYQmR/338-update-organisations-list-page-to-be-inline-with-other-list-pages

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

Follow [these steps](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html) if you are doing a Rails upgrade.

### CODE REVIEW

**Comment by nnagewad** [comment]
Nice!

File: app/controllers/admin/organisations_controller.rb [hunk 1]
```diff
@@ -84,6 +84,8 @@ def update
     end
   end
 
+  def confirm_destroy; end
+
   def destroy
     @organisation.destroy!
     redirect_to admin_organisations_path
```
Comment (chriswhitehouse-tw) [inline app/controllers/admin/organisations_controller.rb:91]: Maybe a notice could be added here to inform the user the organisation has been deleted.

**Review by chriswhitehouse-tw — APPROVED** [review]
@davidgisbey great work, looks good. Only comment on offering the user a notice when an org has been deleted.

File: app/controllers/admin/organisations_controller.rb (hunk 1 above)
Comment (davidgisbey) [reply app/controllers/admin/organisations_controller.rb]: good shout have added it 👍

**Pull request merged**